windows-imager.zig

```diff
@@ -453,6 +516,52 @@ fn imageDisk(disk_handle: win.HANDLE, file_handle: win.HANDLE, file_size: u64, b
     }
 }
 
+fn readDisk(disk_handle: win.HANDLE, file_handle: win.HANDLE, file_size: u64, buf: []u8) !void {
+    std.debug.print("dismounting disk...\n", .{});
+    try dismountDisk(disk_handle);
+    std.debug.print("locking disk...\n", .{});
+    try lockDisk(disk_handle);
+    std.debug.print("disk ready to read\n", .{});
```

Comment:
Do we need to dismount the disk to read it?

Reply:
No, dismounting the disk is not required. It appears that I thought dismounting was a solution for `ACCESS_DENIED` errors while testing on floppies.

That is, in fact, an unrelated error; maybe there should be an error code path for "if device is floppy or some other slow media, wait two seconds if `ACCESS_DENIED` while locking and try again".